cmd/lncli/commands.go

```diff
@@ -2855,11 +2841,10 @@ var listChainTxnsCommand = cli.Command{
 }
 
 func listChainTxns(ctx *cli.Context) error {
-	ctxb := context.Background()
 	client, cleanUp := getClient(ctx)
 	defer cleanUp()
 
-	resp, err := client.GetTransactions(ctxb, &lnrpc.GetTransactionsRequest{})
+	resp, err := client.GetTransactions(getContext(), &lnrpc.GetTransactionsRequest{})
```

Comment:
I also think I would prefer a `ctx := getContext()` at the start of the method, then use that directly here.

Reply:
Done.